src/sdk/go/1/controllers/auth/check-token/snippets/check-token.go

```diff
@@ -5,8 +5,8 @@ res, err := kuzzle.Auth.CheckToken(jwt)
 if err != nil {
   log.Fatal(err)
 } else {
-	if res.Valid != true {
-		log.Fatal("Token not valid")
+  if res.Valid != true {
+		log.Fatal("Invalid token")
```

Comment:
(nitpicking) indentation error 
Don't we have a linter? Or maybe this is correctly indented, but using tabs instead of spaces?